src/main/fc/rc.c

```diff
@@ -572,55 +567,66 @@ FAST_CODE_NOINLINE void calculateFeedforward(const pidRuntime_t *pid, int axis)
     // smooth the setpointSpeed value
     setpointSpeed = prevSetpointSpeed[axis] + pid->feedforwardSmoothFactor * (setpointSpeed - prevSetpointSpeed[axis]);
 
-    // calculate acceleration and attenuate
+    // calculate acceleration from smoothed setpointSpeed and attenuate
     setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * rxRate * 0.01f;
```

Comment:
It isn't obvious where did `0.01f` come from

Reply:
Good point.
It's a 'normalising' factor for the interval.  In the re-factoring I've (maybe) given the rationale.
We currently correct the setpoint speed for higher rates by multiplying by rxRate.  So that if we double the rxRate, the setpointSpeed doesn't change.
For the setpointAcceleration element, we want the smoothed area under the acceleration curve to be the same whether the rxRate is high or low.
If we double the RxRate, compared to the normal rxRate, a step change in setpointSpeed will be 'over' in half the time, and on smoothing will result in half the area under the curve.  So we need to multiply the setpointAcceleration value by a factor that doubles the acceleration element when the RxRate is twice as high as 'normal'.

This 0.01 factor is basically just a 'normalisation' factor for a 'normal' rxRate of 100Hz.
I guess it forms part of the 'strength' of the overall acceleration factor, and it could be included in the initialisation where the `pid->feedforwardBoostFactor` gain  factor is set.  I'll put it there.